package.json

```diff
@@ -1,12 +1,16 @@
 {
+  "private": true,
   "scripts": {
-    "test": "lerna run test",
-    "postinstall": "lerna bootstrap && link-parent-bin"
```

Comment:
Why not hook up on the `postinstall` hook?

Btw, we might not need `lerna bootstrap` anymore if we run `yarn` and have workspaces. We still need `link-parent-bin`, though, since yarn also tends to fail to link bins installed in top-level node_modules.

Reply:
Are you sure yarn fails to link bins? Is there a way to test it?

Reply:
I set-up a few monorepos with yarn workspaces in the last couple of month and encountered that issue at least twice.

Not sure if there is a real way to test. But if a package has a dependency with a CLI tool (`ava` for instance) and the dependency is installed in the monorepo-wide `node_modules`, then I think the tool will always be missing from the package`s `node_modules/.bin` as of now.

Reply:
But the tests are green... 🤔

Reply:
I've just installed this branch and yarn didn't create links for packages that are not in the root, so I guess this is the default behaviour. Why do you need it for by the way?

Reply:
Yarn documentation says:

> `[script]` can also be any locally installed executable that is inside `node_modules/.bin/`.

Reply:
> I've removed the postinstall hook because it causes an infinite loop (lerna/lerna#1125).

Good catch! Question is if we even need `lerna bootstrap` anymore. What's the benefit of `lerna bootstrap` instead of just running `yarn`? It will also install the root dependencies, all package dependencies and link the packages.

Reply:
> Question is if we even need `lerna bootstrap` anymore

So, should I remove the `bootstrap` script from the root package?

Reply:
Let's give it a try :)

(I can do it, too, if you want; in the evening)

Reply:
> Let's give it a try :)

Sorry, @andywer, what do you mean?

Reply:
I meant
> So, should I remove the bootstrap script from the root package?

👉I think it should still work without `lerna bootstrap` when using yarn and yarn workspaces. Let's try and see if that's so 😉

Reply:
Good enough for me 😅

But seriously, I think it does pretty much exactly the same, so let's get rid of one of them.

Reply:
`lerna bootstrap` actually just [uses yarn install](https://github.com/lerna/lerna/issues/1021#issuecomment-333598007) when using workspaces, so +1 to killing any references to it.